src/uikit.cs

```diff
@@ -21156,4 +21200,167 @@ interface UIKey : NSCopying, NSCoding {
 		UIKeyboardHidUsage KeyCode { get; }
 	}
 
+	[NoWatch, NoTV, iOS (13,4)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UIPointerInteraction : UIInteraction {
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		IUIPointerInteractionDelegate Delegate { get; }
+
+		[Export ("enabled")]
+		bool Enabled { [Bind ("isEnabled")] get; set; }
+
+		[Export ("initWithDelegate:")]
+		[DesignatedInitializer]
+		IntPtr Constructor ([NullAllowed] IUIPointerInteractionDelegate @delegate);
+
+		[Export ("invalidate")]
+		void Invalidate ();
+	}
+
+	interface IUIPointerInteractionDelegate { }
+
+	[NoWatch, NoTV, iOS (13,4)]
+	[Protocol, Model (AutoGeneratedName = true)]
+	[BaseType (typeof (NSObject))]
+	interface UIPointerInteractionDelegate {
+
+		[Export ("pointerInteraction:regionForRequest:defaultRegion:")]
+		[return: NullAllowed]
+		UIPointerRegion GetRegionForRequest (UIPointerInteraction interaction, UIPointerRegionRequest request, UIPointerRegion defaultRegion);
```

Comment:
ForRequest looks redundant since the other methods are ForRegion and have a diff signature.

Reply:
mixed emotion: it's a protocol - so it's harder to know if it will conflict/confuse where the interface will be implemented

Reply:
Yeah Sebastien hit the nail ob why I left the name like that